output/schema/schema.json

```diff
@@ -51886,9 +51886,18 @@
     {
       "inherits": [
         {
+          "generics": [
+            {
+              "kind": "instance_of",
+              "type": {
+                "name": "ClusterCertificateInformation",
+                "namespace": "x_pack.ssl.get_certificates"
+              }
+            }
+          ],
           "type": {
-            "name": "ResponseBase",
-            "namespace": "common_abstractions.response"
+            "name": "Array",
+            "namespace": ".Users.pkrauss.github.com.elastic-client-generator.specification.node_modules.typescript.lib"
```

Comment:
What a freaky namespace?! cc @swallez @delvedor any clue how this one made it in ?

Reply:
It's "correct", as `Array` is a native typescript type. I think @philkra wanted to use [`ArrayResponseBase`](https://github.com/elastic/elastic-client-generator/blob/master/docs/behaviors.md#arrayresponsebase).

Reply:
Fixed, @delvedor was on point